packages/edit-site/src/components/editor/index.js

```diff
@@ -50,40 +55,7 @@ const interfaceLabels = {
 	footer: __( 'Editor footer' ),
 };
 
-function useIsSiteEditorLoading() {
```

Comment:
Lifted this state hook into `Layout` because we also need it to enable/disable the resizing feature.